specs-go/state.go

```diff
@@ -9,7 +9,7 @@ type State struct {
 	// Status is the runtime status of the container.
 	Status string `json:"status"`
 	// Pid is the process ID for the container process.
-	Pid int `json:"pid"`
+	Pid *int `json:"pid"`
```

Comment:
I talked to @crosbymichael offline. He's going to carry this. I'll close this PR in favour of #897